### PR TITLE
refactor(@mongodb-js/compass-connect): Refactor file input to be actual file input

### PR DESCRIPTION
**Pull request by gribnoysup**
## Description
Changes file input from using special electron apis to rendering an actual file inpui. From user perspective everything works and looks the same
### Checklist
- [ ] New tests and/or benchmarks are included
- [ ] Documentation is changed or added

## Motivation and Context
This helps with integration tests if we want to test connection with cert files

- [ ] Bugfix
- [ ] New feature
- [ ] Dependency update
- [x] Misc

## Types of changes

- [ ] *Backport Needed*
- [ ] Patch (non-breaking change which fixes an issue)
- [x] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/compass-connect/src/components/form/form-file-input.jsx [hunk 1]
```diff
@@ -2,152 +2,90 @@ import React from 'react';
...
+      />
+      <Button
+        id={id}
+        className={styles['form-item-file-button']}
+        onClick={() => {
+          inputRef.current.click();
+        }}
+        title="Select a file"
+      >
+        <i className="fa fa-upload" aria-hidden />
```
Comment (Anemy) [inline packages/compass-connect/src/components/form/form-file-input.jsx:74]: How do you feel about using `@leafygreen-ui/icon` here? Maybe the `AddFile` https://www.mongodb.design/component/icon/example/ would also be a bit more descriptive than the upload? Also fine how it is, we can address it later as part of the more general overhaul.

**Review by Anemy — APPROVED** [review]
lgtm! Nice on removing the `remote` module - it's deprecated in newer electron releases. 
Manually tested existing connection that had a file and a new connection w/ a file.

File: packages/compass-connect/src/components/form/form-file-input.jsx (hunk 1 above)
Comment (gribnoysup) [reply packages/compass-connect/src/components/form/form-file-input.jsx:74]: Yes, that's a great suggestion, will do!

Comment (gribnoysup) [reply packages/compass-connect/src/components/form/form-file-input.jsx]: Done in f590b4a

**Pull request merged**